Resources/doc/reference/permission_matrix.rst

```diff
@@ -0,0 +1,28 @@
+The permission matrix
+=====================
+
+Configuration
+-------------
+
+The type of form ``sonata_security_roles`` was improved to show all roles in matrix.
+Indeed, for every admin defined in sonata admin we have eight roles by default:
+
+    - Edit
+    - Create
+    - List
+    - Delete
+    - Show
+    - Export
+    - Operator
+    - Master
+
+The permssions are represented in two parts in the following table.
+The first part shows all default permissions for each entity.
+The second part shows all custom defined roles from the ``security.yml`` configuration.
+
+![The permission matrix](../images/permission_matrix.png)
+
+How to use it
+-------------
+
+You need to use the ``sonata_security_roles`` form type.
```

Comment:
New documents need to be added to the table of contents, see https://travis-ci.org/sonata-project/SonataUserBundle/jobs/220280653

Reply:
Do you just update the index.rst to include the new permission_matrix.rst file?
[SonataUserBundle/docs/index.rst](https://github.com/sonata-project/SonataUserBundle/blob/4.x/docs/index.rst)
 with
"reference/permission_matrix"

to fix travis if that helps.